contracts/misc/KyberInternalGovernance.sol

```diff
@@ -0,0 +1,137 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity 0.7.6;
+
+
+import {SafeERC20} from '@openzeppelin/contracts/token/ERC20/SafeERC20.sol';
+import {PermissionAdmin} from '@kyber.network/utils-sc/contracts/PermissionAdmin.sol';
```

Comment:
`PermissionOperators` extends from `PermissionAdmin` so this import is redundant

Reply:
actually you'll need to import both contracts